app/home/-components/support-section/template.hbs

```diff
@@ -7,6 +7,15 @@
             {{t 'new-home.support-section.header'}}
         </h1>
     </header>
+    <div local-class='flex-container'>
+        <div local-class='flex-row iframe-container'>
+            <iframe
+                local-class='osf-video'
+                src='https://www.youtube.com/embed/iebMBpi0prc' 
```

Comment:
Hey Brian, the iframe will need an aria label and alt text. If you have any questions, just let me know.